README.md

```diff
@@ -1625,6 +1625,7 @@ Update Time, five active automations, webhooks.
    * [Brackets](http://brackets.io/) - Brackets is an open-source text editor specifically designed for web development. It is lightweight, easy to use, and highly customizable.
    * [cacher.io](https://www.cacher.io) — Code snippet organizer with labels and support for 100+ programming languages.
    * [Code::Blocks](https://codeblocks.org) — Free Fortran & C/C++ IDE. Open Source and runs on Windows,macOS & Linux.
+   * [Codelobster](https://codelobster.com/) - Codelobster is an smart free code editor with many plugins for all popular frameworks.
```

Comment:
Free seems quite limited in supported frameworks and languages, please mention what it supports https://codelobster.com/order-ide.php